modules/tinymce/changelog.txt

```diff
@@ -1,4 +1,5 @@
 Version 5.5.0 (TBD)
+    Added function to editor to determine if a plugin exists or not #TINY-766
```

Comment:
We should mention what the function is here. eg
```
Added `hasPlugin` function to the editor API to determine if a plugin exists or not #TINY-766
```

Reply:
When you put it like that...